src/generators/building-rollup/templates/package.json

```diff
@@ -4,17 +4,16 @@
     "start:build": "web-dev-server --root-dir dist --app-index index.html --open"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.16.11",
-    "@open-wc/building-rollup": "^2.0.2",
-    "@rollup/plugin-babel": "^5.3.1",
-    "@rollup/plugin-node-resolve": "^13.0.6",
-    "@web/rollup-plugin-html": "^1.11.1",
-    "@web/rollup-plugin-import-meta-assets": "^1.0.8",
+    "@babel/preset-env": "^7.24.6",
```

Comment:
do we even need babel/preset-env anymore? Don't we use the esbuild plugin for this?

Reply:
It looks like its not used in the rollup config at all anymore, only babel-plugin-template-html-minifier is used, so I guess we can remove it

Reply:
Nice call!! I'll get on that.